src/tools/spin/spinquic.cpp

```diff
@@ -1074,6 +1078,7 @@ main(int argc, char **argv)
         PrintHelpText();
     }
 
+    CxPlatLockInitialize(&RunThreadLock);
```

Comment:
Do this after initializing the platform. Also, please uninitialize it when done.

Reply:
Thanks, fixed